File: ThreadsApp/ThreadsApp/View/About/AboutViewController.m
//
//  AboutViewController.m
//  ThreadsApp
//
//  Created by Brooma Service on 18/04/17.
//  Copyright © 2017 Brooma Service. All rights reserved.
//

#import "AboutViewController.h"

@interface AboutViewController ()

@end

@implementation AboutViewController

#pragma mark - Life Cycle

- (void)viewDidLoad {
    [super viewDidLoad];
    self.navigationItem.title = NSLocalizedString(@"unreaded_messages_test", nil);
    self.titleLabel.text = NSLocalizedString(@"unreaded_messages", nil);
    self.counterLabel.text = @"0";
}

@end

File: ThreadsApp/ThreadsAppSwift/View/About/AboutViewController.swift
//
//  AboutViewController.swift
//  THRAppSwift-Polarbear
//
//  Created by Brooma Service on 14/05/2019.
//  Copyright © 2019 Brooma Service. All rights reserved.
//

import Foundation
import UIKit

class AboutViewController : UIViewController {
    
    @IBOutlet weak var titleLabel: UILabel!
    @IBOutlet weak var counterLabel: UILabel!
    private var counterValue: Int = 0;
    
    // MARK: - Life Cycle
    
    override func viewDidLoad() {
        
        super.viewDidLoad()
        
        self.navigationItem.title = NSLocalizedString("unreaded_messages_test", comment:"");
        self.titleLabel.text = NSLocalizedString("unreaded_messages", comment:"");
        self.updateCouterLabel()
    }
    
    func setCounterValue(_ counterValue: Int) {
        self.counterValue = counterValue;
        self.updateCouterLabel()
        
    }
    
    func updateCouterLabel() {
        self.counterLabel?.text = String(format: "%ld", self.counterValue)
    }
}
